src/views/InvolvementPanel/InvolvementPanel.js

```diff
@@ -18,6 +19,10 @@ export default class InvolvementPanel extends React.Component {
         text: 'Volunteer',
         view: <VolunteerFormView />
       },
+      TEACHER: {
```

Comment:
It reduces fine in Google Chrome and Microsoft Edge. Are you aware that the text of the Get Involved page does not display properly in Internet Explorer? Not just in this branch but on the current website right now.
![screenshot 13](https://user-images.githubusercontent.com/9382984/29009075-ffe9da1c-7aed-11e7-8028-86544e5b0e07.png)